.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: required
 dist: trusty
+group: travis_lts
```

Comment:
Do we really want to do this? It could lead to us having to tackle far more issues when they do upgrade the LTS images rather than dealing with them little at a time as they come

Reply:
`travis_lts` is the default value if we don't set this, so I was just making it explicit.
The only alternative is setting `travis_latest`. Do you think we should use that instead?

Reply:
I'm fine with leaving it out and just going with the default behavior so we're on the "paved path"

Reply:
Removed this commit